FV3/atmos_cubed_sphere/tools/fv_grid_tools.F90

```diff
@@ -890,11 +920,11 @@ subroutine init_grid(Atm, grid_name, grid_file, npx, npy, npz, ndims, nregions,
           dyc(i,jed+1) = dyc(i,jed)
        end do
 
-
```

Comment:
Just maybe undo this and the rest of the spacing changes in this file for a cleaner history?

Reply:
ok